packages/video_player/tizen/src/video_player.cc

```diff
@@ -422,6 +435,26 @@ void VideoPlayer::SendInitialized() {
   }
 }
 
+Eina_Bool VideoPlayer::ResetScreensaverTimeout(void *data) {
+  LOG_DEBUG("[VideoPlayer] Reset screen saver timeout.");
+
+  auto *player = static_cast<VideoPlayer *>(data);
+  if (!player->screensaver_reset_timeout_) {
+    LOG_INFO("[VideoPlayer] Symbol not found: %s", dlerror());
+    dlclose(player->screensaver_handle_);
+    return ECORE_CALLBACK_CANCEL;
+  }
+
+  int ret = player->screensaver_reset_timeout_();
+  if (ret != 0) {
+    LOG_INFO("screensaver_reset_timeout failed", get_error_message(ret));
+    dlclose(player->screensaver_handle_);
```

Comment:
Closing the handle here seems dangerous (calling `screensaver_reset_timeout` after dlclose is undefined). The handle will be closed by `Dispose`.

Reply:
Fixed.